lightkurve/targetpixelfile.py

```diff
@@ -684,6 +684,216 @@ def update(cadence, log_stretch):
+            # Each data source will later become a hover-over tooltip
+            source = ColumnDataSource(data=dict(
+                time=lc.time,
+                time_iso=human_time,
+                flux=lc.flux,
+                cadence=lc.cadenceno,
+                quality_code=lc.quality,
+                quality=np.array(qual_strings)))
+
+            # Provide extra metadata in the title
+            if self.mission == 'K2':
+                title = "Quicklook lightcurve for EPIC {} (K2 Campaign {})".format(
+                    self.keplerid, self.campaign)
+            elif self.mission == 'Kepler':
+                title = "Quicklook lightcurve for KIC {} (Kepler Quarter {})".format(
+                    self.keplerid, self.quarter)
+            elif self.mission == 'TESS':
+                title = "Quicklook lightcurve for TIC {} (TESS Sector {})".format(
+                    self.ticid, self.sector)
+            else:
+                title = "Quicklook lightcurve for target {}".format(self.targetid)
+
+            ymax = np.nanpercentile(self.to_lightcurve(aperture_mask='all').flux, 80)*1.2
+            fig1 = figure(title=title, plot_height=300, plot_width=600, y_range=(0, ymax),
+                       tools="pan,wheel_zoom,box_zoom,reset")#, theme=theme)
+            fig1.yaxis.axis_label = 'Normalized Flux'
+            fig1.xaxis.axis_label = 'Time - 2454833 (days)'
+            step_dat = fig1.step('time', 'flux', line_width=1, color='gray', source=source, nonselection_line_color='gray')
+
+            r = fig1.circle('time', 'flux', source=source, fill_alpha=0.3, size=8,line_color=None,
+                         selection_color="firebrick", nonselection_fill_alpha=0.0,
+                         nonselection_fill_color="grey",nonselection_line_color=None,
+                         nonselection_line_alpha=0.0, fill_color=None,
+                         hover_fill_color="firebrick",hover_alpha=0.9,hover_line_color="white")
+
+            fig1.add_tools(HoverTool(tooltips=[("Cadence", "@cadence"),
+                                               ("Time ({})".format(lc.time_format.upper()), "@time{0,0.000}"),
+                                               ("Time (ISO)", "@time_iso"),
+                                               ("Flux", "@flux"),
+                                               ("Quality Code", "@quality_code"),
+                                               ("Quality Flag", "@quality")],
+                                     renderers=[r],
+                                     mode='mouse',
+                                     point_policy="snap_to_data"))
+            # Vertical line to indicate the cadence shown in Fig 2
+            vert = Span(location=0, dimension='height', line_color='firebrick',
+                        line_width=4, line_alpha=0.5)
+            fig1.add_layout(vert)
+
+
+            fig2 = figure(plot_width=300, plot_height=300, x_range=(min(xx), max(xx+1)),
```

Comment:
The code below can be clarified by calling this e.g. `fig_pixels` instead of `fig2`.   Likewise, I'd use e.g. `fig_lc` instead of `fig1`.